src/Assignment.ts

```diff
@@ -0,0 +1,59 @@
+import Question from "./model/Question";
+import alasql from "alasql";
+
+const question = new Question(
+    // Prompt for the user
+    `Select all columns from the "Employees" table.`,
+
+    // SQL queries to configure the context
+    `CREATE TABLE Employees (ID INT, Name VARCHAR(255), Supervisor INT);
+    INSERT INTO Employees VALUES
+        (1, "Joe", NULL),
+        (2, "Bill", 1),
+        (3, "Jill", NULL),
+        (4, "Bob", 3),
+        (5, "Marvin", 1);`,
+
+    // Answer key query
+    `SELECT * FROM Employees`);
+
+function compareResults(a: Array<any>, b: Array<any>) {
+    if (a.length != b.length) return false;
+
+    // TODO: This isn't ideal, but it gets the job done for the time being. Write something better in the future.
```

Comment:
I'm not a big fan of TODO comments being merged into the dev or main branches. Shipped code should work as is and not include these kinds of comments because they often stick around way longer than they should and it becomes confusing as the product evolves and the intent/plan the original developer had in mind is often lost. Better to track things like this elsewhere, ideally as work items that can have more context and clearly define the goal.

Schools always teach you to use a lot of comments but in the real world comments should be viewed as a necessary evil. Your code should be clear enough  that they aren't necessary. Comments are only for when you are not able to express your intent or be clear with just code for whatever reason.